eda_analysis/eda_analysis.py

```diff
@@ -145,4 +149,47 @@ def calc_cor(dataframe, num_vars):
     >>>num_var = ['height', 'width']
     >>> calc_cor(X, num_vars)    
     """
-    # Code
\ No newline at end of file
+    # Test input 'dataframe' is a dataframe
+    assert isinstance(dataframe, pd.DataFrame), "Input 'dataframe' is not a dataframe"
```

Comment:
```suggestion
      if not isinstance(dataframe, pd.DataFrame):
          raise Exception("Input 'dataframe' is not a dataframe")
```

Reply:
as @marvinmin said, assert is not a good catch-all for production code, since certain intepreter settings can skip assert statements entirely.

Reply:
Changed in the latest commit.